code/__HELPERS/_lists.dm

```diff
@@ -9,6 +9,14 @@
  * Misc
  */
 
+// Generic listoflist safe add and removal macros:
+///If value is a list, wrap it in a list so it can be used with list add/remove operations
+#define LIST_VALUE_WRAP_LISTS(value) (islist(value) ? list(value) : value)
```

Comment:
am I stupid, or would 
```suggestion
#define LIST_VALUE_WRAP_LISTS(value) (list(value))
```
not work here? guess it depends on how cheap the islist check is

Reply:
> am I stupid, or would
> 
> not work here? guess it depends on how cheap the islist check is

islist() is a built-in that compares type ids i assume, which would make it far faster than instantiating a new object type (list)